examples/lang/hetzner_vm0.mcl

```diff
@@ -0,0 +1,42 @@
+import "os"
+#import "deploy"
+
+# read token from a local path outside repo:
+$f = "../mgmt-dev/apitoken.txt"
+$token = os.readfile($f)
+
+# alternatively, read from the deploy:
+# $token = deploy.readfile(...)
+
+# or give the string directly:
+# $token = "..."
+
+hetzner:server "resourcedemo" {
```

Comment:
I eventually decided on hetzner:vm, although I'm still not fully convinced that one is better than the other. In terms of future-proofing the naming convention, I think that it might be better to use something like `hetzner:hcloud:server`. It names the provider, the service, and the resource type. But I also understand that this becomes a bit bulky. 

What do you think now that you've had some time to think about it? Maybe it's easier to settle this in a quick call?